Resources/Textures/Objects/Weapons/Guns/Shotguns/flaregun_security.rsi/meta.json

```diff
@@ -0,0 +1,33 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from cev-eris at https://github.com/discordia-space/CEV-Eris/raw/3f9ebb72931ff884427c3004a594ec61aaaa7041/icons/obj/guns/projectile/flaregun.dmi and edited",
```

Comment:
Eris textures are usually not compatible with ours, this will need art director approval

Reply:
This is the exact texture used by the current ingame flaregun, just with the orange pixels changed out for red ones, let me know if that is still a problem.

Reply:
The sprite has been OK'ed by [one of the art leads](https://discord.com/channels/310555209753690112/346743671787356160/1299902662384554044)